roles/trento_rpms/tasks/main.yml

```diff
@@ -0,0 +1,47 @@
+# code: language=ansible
+---
+- name: Create secrets
+  no_log: true
+  ansible.builtin.set_fact: # noqa: var-naming[no-jinja]
+    "{{ item }}": "{{ lookup('community.general.random_string', base64=True, length=64) }}"
+  when: lookup('vars', item) == ""
+  loop:
+    - secret_key_base
+    - access_token_secret
+    - refresh_token_secret
```

Comment:
I guess we should do the same variable checking for SMPT we do for the containers here, before creating the config files